pybikes/metro_bike_share.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+class MetroBikeShare(BikeShareSystem):
+
+    sync = True
+
+    meta = {
+        'system': 'Metro Bike Share',
+        'company': ['Los Angeles County Metropolitan Transportation Authority (Metro)', 'Bicycle Transit Systems Inc', 'B-cycle'],
+    }
+
+    def __init__(self, tag, feed_url, meta):
+        super(MetroBikeShare, self).__init__(tag, meta)
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+            scraper.ssl_verification = False
```

Comment:
Otherwise I got that SSL Error:

```
  File "pybikes/metro_bike_share.py", line 25, in update
    data = json.loads(scraper.request(self.feed_url))
  File "pybikes/utils.py", line 65, in request
    verify=self.ssl_verification,
  File "/usr/local/lib/python2.7/dist-packages/requests-2.0.0-py2.7.egg/requests/sessions.py", line 357, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests-2.0.0-py2.7.egg/requests/sessions.py", line 460, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests-2.0.0-py2.7.egg/requests/adapters.py", line 358, in send
    raise SSLError(e)
requests.exceptions.SSLError: hostname 'bikeshare.metro.net' doesn't match either of '*.wpengine.com', 'wpengine.com'
```